src/components/floweditor/FlowEditor.tsx

```diff
@@ -206,11 +206,6 @@ export const FlowEditor = () => {
       Track('Flow opened');
 
       return () => {
-        Object.keys(files).forEach((node: any) => {
```

Comment:
@mdshamoon 

It would be good to get your thoughts on this.

Reply:
When we are loading the floweditor it adds all the js and css files from the build folder to the document body. On unmounting the component, we are just removing those css and js files. the files object holds the node information.

If you inspect the document you will see these files appended at the end of the body.

If we remove this logic then everytime it adds the same set of files to the body without removing the previous ones.

This could only fail in the case when the node is not present in the body. I think we can add a check for that.

Reply:
fixed this